prow/jobs.yaml

```diff
@@ -185,3 +185,23 @@ kubernetes/test-infra:
   context: Jenkins go test
   rerun_command: "@k8s-bot go test this"
   trigger: "@k8s-bot (go )?test this"
+  spec:
```

Comment:
Agreed, I'm going to make some changes to cut this down to pretty much nothing. For now this is what we've got.